apps/dashboard/src/components/auth/inbox-playground.tsx

```diff
@@ -102,7 +102,7 @@ export function InboxPlayground() {
   }, [telemetry]);
 
   useEffect(() => {
-    if (!data) return;
+    if (!data || isLoading) return;
```

Comment:
Isn't the lack of data the same `isLoading` in the react-query kingdom?

Reply:
Yep you are correct, my issue was actually from something else. Investigating